src/main/osd/osd_elements.c

```diff
@@ -922,6 +922,18 @@ static void osdElementGpsSpeed(osdElementParms_t *element)
 {
     tfp_sprintf(element->buff, "%c%3d%c", SYM_SPEED, osdGetSpeedToSelectedUnit(gpsConfig()->gps_use_3d_speed ? gpsSol.speed3d : gpsSol.groundSpeed), osdGetSpeedToSelectedUnitSymbol());
 }
+
+static void osdElementEfficiency(osdElementParms_t *element)
+{
+    const int32_t speed = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed);
+    if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX) && speed > 0) {
+        const int32_t mA = getAmperage() * 10;
+        const int32_t efficiency = mA / speed;
+        const char *unitSymbol = osdConfig()->units == OSD_UNIT_IMPERIAL ? "MI" : "KM";
```

Comment:
Use the symbols `SYM_MILES` and `SYM_KM`. Also change the print formatting to use `%c` for the symbol.

Reply:
Since the `SYM_MILES` and `SYM_KM` symbols include the `/h`, that would result in e.g. `123 mAh/km/h`

Reply:
The `SYM_MILES` (0x7E) and `SYM_KM` (0x7D) don't include the "/h". The are just "mi" and "km" respectively. Maybe you need to update the font in your OSD chip. "MPH" and KPH" are `SYM_MPH` and `SYM_KPH` respectively.

![Screen Shot 2020-03-17 at 6 06 45 PM](https://user-images.githubusercontent.com/17088539/76906409-4324b780-687a-11ea-9a17-cd81c97af6e2.png)

Reply:
Right you are, not sure how I got confused there.  Thanks!